src/Uno.UWP/Devices/Geolocation/Geolocator.Android.cs

```diff
@@ -397,6 +401,7 @@ private void RemoveUpdates()
 			&& _locationListener.Handle != IntPtr.Zero)
 		{
 			_locationManager.RemoveUpdates(_locationListener);
+			BroadcastStatusChanged(PositionStatus.Disabled);
```

Comment:
I'm not sure about this change. Why it is needed?

Reply:
We don't update the localisation anymore a this point. It's seem weird when Status Changed is attached and when we Disabled the localisation that it's not updated. 

If unnacessary I can removed this changed

Reply:
RemoveUpdates() is called in many places, not sure if that is the right choice?
What you could do is to BroadcastStatusChanged(PositionStatus.Disabled) in StopPositionChanged()